src/puppetlabs/trapperkeeper/services/webserver/experimental/jetty9_websockets.clj

```diff
@@ -50,13 +51,16 @@
     (.. this (getSession) (getUpgradeRequest) (isSecure)))
```

Comment:
I had something here before, but decided to remove it to avoid changing behavior. It wasn't really needed.